### PR TITLE
Feature/create component

### PR DESCRIPTION
**Pull request by jsconan**
Move the diagnostic tools into a standalone component to be able to integrate it in other views.

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -140,5 +140,9 @@ public function update($initialVersion)
 
             $this->setVersion('1.5.0');
         }
+
+        if ($this->isVersion('1.5.0')) {
+            $this->setVersion('1.6.0');
+        }
```
Comment (krampstudio) [inline scripts/update/Updater.php]: Shouldn't you use `$this->skipVersion('1.5.0', '1.6.0');` (_I'm not sure about the exact syntax_) ?

File: views/js/controller/Authenticator/message.js [hunk 2]
```diff
@@ -13,37 +13,33 @@
...
+    return {
+        start: function() {
+            var $feedbackBox = $('#feedback-box');
+
+            if(!!$feedbackBox.data('error')){
+                feedback().error($feedbackBox.data('error'));
+            }
+            if($feedbackBox.data('message')){
+                feedback().error($feedbackBox.data('message'));
+            }
```
Comment (krampstudio) [inline views/js/controller/Authenticator/message.js]: It seems this source code is repeated in the other controller, could you wrap it  and call it in both controllers ?

**Comment by jsconan** [comment]
Changes made

**Comment by krampstudio** [comment]
Linked to TAO-2166

**Pull request merged**